react/features/filmstrip/actions.web.js

```diff
@@ -80,8 +86,41 @@ export function setVerticalViewDimensions() {
     return (dispatch: Dispatch<any>, getState: Function) => {
         const state = getState();
         const { clientHeight = 0, clientWidth = 0 } = state['features/base/responsive-ui'];
+        const { width: filmstripWidth } = state['features/filmstrip'];
         const disableSelfView = shouldHideSelfView(state);
-        const thumbnails = calculateThumbnailSizeForVerticalView(clientWidth);
+        const resizableFilmstrip = isFilmstripResizable(state);
+        const thumbnails = resizableFilmstrip
+            ? calculateThumbnailSizeForResizableVerticalView(clientWidth, filmstripWidth.current ?? 0)
```

Comment:
Why not just put a default value `0` for the `filmstripWidth` in `calculateThumbnailSizeForResizableVerticalView`?